### PR TITLE
Ask for company name/size or goal when asking about GDevelop usage

### PR DESCRIPTION
**Pull request by 4ian**
http://gdevelop-storybook.s3-website-us-east-1.amazonaws.com/feature/company-and-goal/latest/index.html?path=/story/profile-additionaluserinfodialog--default

image
image
image

### CODE REVIEW

File: newIDE/app/src/Profile/AdditionalUserInfoDialog.js [hunk 1]
```diff
@@ -108,20 +109,36 @@ const hearFromOptions = [
...
-    title: <Trans>Have you used other game engines?</Trans>,
-    options: creationExperienceOptions,
-  },
-  hearFrom: {
-    title: <Trans>How did you hear about GDevelop?</Trans>,
-    options: hearFromOptions,
+const teacherOrTrainingCreationGoalOptions = [
+  {
+    value: 'teach-students',
+    label: `Teach students the principles of Game Development and logic`,
```
Comment (ClementPasteau) [inline newIDE/app/src/Profile/AdditionalUserInfoDialog.js:115]: your labels have unnecessary backticks

Comment (ClementPasteau) [reply newIDE/app/src/Profile/AdditionalUserInfoDialog.js:115]: or you forgot to use `t`

File: newIDE/app/src/Profile/AdditionalUserInfoDialog.js [hunk 2]
```diff
@@ -108,20 +109,36 @@ const hearFromOptions = [
...
   },
-};
+  { value: 'train-workers', label: `Train workers on specific scenarios` },
+  { value: 'therapy', label: `Develop tools for therapy or rehab` },
```
Comment (ClementPasteau) [inline newIDE/app/src/Profile/AdditionalUserInfoDialog.js:118]: That seems quite specific. 🤔 
Should it be more about training the brain?

**Review by ClementPasteau — APPROVED** [review]
lgtm with a few comments

File: newIDE/app/src/Profile/AdditionalUserInfoDialog.js (hunk 1 above)
Comment (4ian) [reply newIDE/app/src/Profile/AdditionalUserInfoDialog.js:115]: I forgot the t everywhere

File: newIDE/app/src/Profile/AdditionalUserInfoDialog.js (hunk 2 above)
Comment (4ian) [reply newIDE/app/src/Profile/AdditionalUserInfoDialog.js]: Tagging @LuniMoon to see if we should change this, in the meantime I'll go ahead with this :)

**Pull request merged**